src/Swarm/Doc/Pedagogy.hs

```diff
@@ -87,16 +90,16 @@ extractCommandUsages idx siPair@(s, _si) =
 -- | Obtain the set of all commands mentioned by
 -- name in the tutorial's goal descriptions.
 getDescCommands :: Scenario -> Set Const
-getDescCommands s =
-  S.fromList $ mapMaybe (`M.lookup` txtLookups) backtickedWords
+getDescCommands s = S.fromList $ concatMap filterConst allCode
  where
   goalTextParagraphs = concatMap (view objectiveGoal) $ view scenarioObjectives s
-  allWords = concatMap (T.words . T.toLower) goalTextParagraphs
-  getBackticked = T.stripPrefix "`" <=< T.stripSuffix "`"
-  backtickedWords = mapMaybe getBackticked allWords
-
-  commandConsts = filter isConsidered allConst
-  txtLookups = M.fromList $ map (syntax . constInfo &&& id) commandConsts
+  allCode = concatMap findCode goalTextParagraphs
+  filterConst :: Syntax -> [Const]
+  filterConst sx = mapMaybe toConst $ universe (sx ^. sTerm)
```

Comment:
@kostmo this is one of the little nice things a parsed code gives us. 🤩 

Hopefully `universe` works just as well as your original implementation, but please check that I did not break it. 😅